data-prepper-plugins/s3-source/build.gradle

```diff
@@ -31,6 +31,11 @@ dependencies {
     testImplementation 'com.github.tomakehurst:wiremock:3.0.0-beta-5'
     testImplementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
     testImplementation project(':data-prepper-test-common')
+    testImplementation project(':data-prepper-plugins:parquet-codecs')
+    testImplementation project(':data-prepper-plugins:csv-processor')
+    testImplementation project(':data-prepper-plugins:parse-json-processor')
+    testImplementation project(':data-prepper-plugins:newline-codecs')
+    testImplementation project(':data-prepper-plugins:avro-codecs')
```

Comment:
This is now in a different PR. Please remove this line so that the build passes.